Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# syntax=docker/dockerfile:1
+
+######################### BUILD STAGE 1 ############################
+# This stage builds the HarbourBridge binary using its source code.
+
+# Add the official Docker golang image as the base image for the build stage
+FROM golang:latest AS build
+
+# Set the default work directory as “/hbBin” in the container.
+WORKDIR /hbBin
+
+# Copy HarbourBrdige’s source code into the working directory.
+COPY . ./
+# Download HarbourBridge’s dependencies mentioned in the go.mod file.
+RUN go mod tidy 
+# Compile the source code and store the binary in the work directory /hbBin.
+RUN make build-static  
+
+######################### Release Stage ###########################
+# This stage consists of the GCloud SDK and the HabrbourBridge binary.
+
+# Add the Ubuntu base image from the Google Container Registry.
+FROM marketplace.gcr.io/google/ubuntu1804:latest
+# Set the default working directory.
+# This directory binds with the host directory that is to store the output files generated by HarbourBridge.
+WORKDIR /harbourBridge/harbour_bridge_output
```

Comment:
nit: consider using harbourbridge everywhere instead of harbourBridge for consistency

Reply:
Also, to clarify, we don't need this step right? harbourbridge should have been able to create this automatically to right?

Reply:
No, it is needed. In the present code, HarbourBridge puts the output files in the location from where the migration commands are called. So we've set the point to call the migration command from - the default working directory as `/harbourBridge/harbour_bridge_output`. This is where the output files will be stored after migration. This directory will be bind mounted with the host's file system to put the output files in the latter.

Reply:
If we change `harbourBridge` to `harbourbridge`, the file paths for the binary file `harbourbridge` will become slightly less clear. eg - `harbourbridge/bin/harbourbridge` in the release stage, `harbourbridge/harbourbridge` in the build stage. Because of this reason we've used `harbourBridge` for the directory names and `harbourbridge` for the binary file name.

Reply:
harbourbridge/bin/harbourbridge should be fine, if you see, even mysql uses mysq/bin/mysql.
hbBin we can keep as is since it is temporary. My concern is if the user logs in to the container, they will finally see the conventional productname/bin/productname rather than different casing at each level.

Reply:
Done.